### ISSUE TITLE
issue on grad

### ISSUE THREAD

**Issue by Lijin002** [Sunnydreamrain/IndRNN_pytorch#12]
Hi, thanks for your great work! But I have encountered something wrong while reproducing the code. 
During "train", the grad of the most layers is "None".  Except layers "classify_weight"、"classify_bias"、"RNN5_weight"、"RNN5_bias" have not-None grad, others have grad which are None. As a result, error happens when running to "grad_climp", as showed in the following figure.
Maybe something goes wrong with layer "RNN5_weight_hh" during loss.backward() I think.
![lossNone_issue](https://github.com/Sunnydreamrain/IndRNN_pytorch/assets/94791260/dbc06e8c-62e4-4390-a1ac-d1b616eaee14)

I wonder how to address this problem. Looking forward to the reply, thank you!
